index.html

```diff
@@ -526,14 +526,15 @@ <h4>Script Type</h4>
           </pre>
         </section>
         <section>
-          <h4>Primary Language</h4>
+          <h4>Languages</h4>
           <p>The <dfn>Primary Language</dfn> is a mandatory property of a <a>DAPT Script</a>
             which represents the default language for the <a>Text</a> content of <a>Script Events</a>.
             This language may be the <a>Original</a> language or a <a>Translation</a> language.
             When it represents a <a>Translation</a> language, it may be the final language
             for which a dubbing or audio description <a>script</a> is being prepared,
             called the <dfn>Target Recording Language</dfn> or it may be an intermediate, or pivot, language
-             used in the workflow.
+             used in the workflow. The <dfn>Original Language</dfn> is an optional property of a <a>DAPT Script</a>
```

Comment:
I think it'd be clearer to have this in a separate paragraph.

```suggestion
            used in the workflow.
          </p>
          <p>The <dfn>Original Language</dfn> is an optional property of a <a>DAPT Script</a>
```